tsconfig.json

```diff
@@ -69,6 +69,6 @@
     "skipLibCheck": true                            /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": ["src", "stories"],
+  "include": ["src", "stories", "Card.stories.tsx"],
```

Comment:
hmmmm why?

Reply:
I have no idea, but remove with [620fbfc](https://github.com/Major-Eth/yearn-web-lib/pull/27/commits/620fbfceb33ad4f0d1c9b54d0db59f9342d04946)

Reply:
Hahaha 😆 all good, yeah, I don't think that should be needed at all 👍